Platform/obc/hal/comms_hal.c

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright (C) 2015  University of Alberta
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+/**
+ * @file comms_hal.c
+ * @author Arash Yazdani
+ * @date 2020-10-01
+ */
+
+/*
+ * When TRX connected, the stubbed blocks can be used for TRX = off situation.
+ */
+
+
+#include "obc_hal.h"
+#include "comms_hal.h"
```

Comment:
I'd rename these files to `s_hal` but last time I did a similar thing, I had to rebuild my whole CCS project. How would you prevent that?

Reply:
I'm not sure what you mean, like you had to recompile?

Reply:
No, the project didn't build anymore. But, it was the name of the directory. I guess it won't be a problem. I'll just give it a try.

Reply:
Yeah if it was in a new place you might have to play with the include directories in the project properties